packages/lib/package.json

```diff
@@ -19,6 +19,8 @@
     "module": "dist/index.js",
     "types": "dist/index.d.ts",
     "exports": {
+      "./dist/style.css": "./dist/style.css",
+      "./style.css": "./dist/style.css",
```

Comment:
Those were already present in `@h5web/app`'s `package.json`. I must have forgotten to add them here too. This is why the Vite codesandbox doesn't quite work yet. I figure we can still add the link in the package's README, since I'll upgrade the sandboxes after the next release.